travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -2,16 +2,21 @@
 #  pylint configuration file will display them without affecting your build status.
 
 # Beta message and code:
+#   api-one-deprecated - W8104
```

Comment:
A similar case with: [CONTRIBUTING.md#idioms](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms)
`Exceptions: Use from openerp.exceptions import Warning as UserError (v8) or from openerp.exceptions import UserError (v9) or find a more appropriate exception in openerp.exceptions.py`